kgitb/rules/__init__.py

```diff
@@ -31,14 +31,15 @@
 def _get_rules(module):
     """Get all functions from a module."""
     return [tup[1] for tup in getmembers(module, isfunction)
-            if tup[0][0] != "_"]
+            if tup[0][0] != '_']
+
 
 _line_rules = _get_rules(line_rules)
 _status_rules = _get_rules(status_rules)
 
 
 def _is_merge(status_line):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a docstring to this function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=python%3AS1720)